docs/logging-handlers-GAEHandler.rst

```diff
@@ -0,0 +1,6 @@
+Google App Engine Flexible Log Handler
+=======================================
+
+.. automodule:: google.cloud.logging.handlers.GAEHandler
```

Comment:
Bad module name, [PEP8 says][1]

> Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.

[1]: https://www.python.org/dev/peps/pep-0008/#package-and-module-names